README.md

```diff
@@ -1,5 +1,11 @@
 # Smart Contract init
-Service Will initialize Streamrs Smart Contracts
+Service Will initialize Streamr's Smart Contracts
+
+smart-contracts-init container is no longer part of streamr-docker-dev. Instead we use the docker-compose.yml file here to build the preloaded parity images streamr/open-ethereum-poa-mainchain-preload1 and streamr/open-ethereum-poa-sidechain-preload1.
+
+To build images:
```

Comment:
In what situation would you do this? When building the images for Docker Hub? That process should be automated in a script IMO, we can't rely on hand-built images if there's a need to change/fix something later.

Reply:
It is automated by preload_parity_images.sh. I'm not sure how the nightly builds are invoked. @mikhaelsantos ?